prow/plugins/cat/cat.go

```diff
@@ -126,24 +126,24 @@ func (cr catResult) Format() (string, error) {
 	return fmt.Sprintf("[![cat image](%s)](%s)", img, src), nil
 }
 
-func (r *realClowder) Url(category string, movieCat bool) string {
-	r.lock.RLock()
-	defer r.lock.RUnlock()
-	uri := string(r.url)
+func (c *realClowder) URL(category string, movieCat bool) string {
```

Comment:
This doesn't need a godoc?

Reply:
`golint` doesn't complain about this one for some reason.